data/abilities/lateral-movement/c5e75d70-09ae-469f-8717-621a1b7ef946.yml

```diff
@@ -11,7 +11,7 @@
     windows:
       psh:
         command: |
-          $username = "#{comp.user.name}";
+          $username = "#{domain.user.name}";
           $password = "#{comp.user.password}";
```

Comment:
there should be parity here, if username is domain then so should the password

Reply:
This probably wasn't the best way to do it, but I made a separate branch/PR for each term. So the username and the password do both match, I just made the respective changes across two pull requests. The "Changing comp.user.password to either host.x.x or domain.x.x dependin…" PR has this change